apollo-compiler/src/main/kotlin/com/apollostack/compiler/GraphqlCompiler.kt

```diff
@@ -28,12 +28,9 @@ open class GraphqlCompiler {
 
   private fun String.formatPackageName(): String {
     val parts = split(File.separatorChar)
-
-    for (i in 2..parts.size) {
-      if (parts[i - 2] == "src" && parts[i] == "graphql") {
-        return parts.subList(i + 1, parts.size).dropLast(1).joinToString(".")
-      }
-    }
+    (2..parts.size)
```

Comment:
@sav007 turns out this is the code you had originally haha, Android Studio suggested the auto refactoring